rlclientlib/serialization/fb_serializer.h

```diff
@@ -33,7 +33,7 @@ namespace reinforcement_learning { namespace logger {
     }
 
     static int serialize(ranking_event& evt, flatbuffers::FlatBufferBuilder& builder,
-                         flatbuffers::Offset<fb_event_t>& ret_val, api_status* status) {
+                         flatbuffers::Offset<fb_event_t>& ret_val, const char* app_id, api_status* status) {
```

Comment:
This is not how event data is passed down to this method, it must be part of ranking_event.

Reply:
Moved it to generic_event. Need to only update v2 schema in this pr.